homeassistant/components/unifi/config_flow.py

```diff
@@ -112,15 +122,17 @@ async def async_step_user(self, user_input=None):
                 )
                 return self.async_abort(reason="unknown")
 
-        host = ""
-        if await async_discover_unifi(self.hass):
+        host = self.config.get(CONF_HOST)
```

Comment:
```suggestion
        host = self.config.get(CONF_HOST, "")
```

Reply:
If voluptuous and gui doesn't care it is fine to leave it as is

Reply:
`default=None` should be fine here for `vol`